src/features/commands.ts

```diff
@@ -672,48 +679,74 @@ Instead:
   },
   {
     words: ["!lock"],
-    help: "",
-    category: "Communication",
+    help: "used for locking channel",
+    category: "Moderation",
     handleMessage: async (msg) => {
-      if (!msg.guild || !isStaff(msg.member)) {
-        return;
-      }
-
-      // permission overwrites can only be applied on Guild Text Channels
-      if (msg.channel.type === "GUILD_TEXT") {
-        const { channel: guildTextChannel } = msg;
-        await guildTextChannel.permissionOverwrites.create(
-          guildTextChannel.guild.roles.everyone,
-          {
-            SEND_MESSAGES: false,
-            CREATE_PUBLIC_THREADS: false,
-            CREATE_PRIVATE_THREADS: false,
-            SEND_MESSAGES_IN_THREADS: false,
-          },
-        );
-        guildTextChannel.send("This channel has been locked by a moderator");
-      }
+      if (!msg.guild || !isStaff(msg.member)) return
+        // permission overwrites can only be applied on Guild Text Channels
+        if (msg.channel.type === "GUILD_TEXT") {
+          const { channel: guildTextChannel } = msg;
+          await guildTextChannel.permissionOverwrites.create(
+            guildTextChannel.guild.roles.everyone,
+            {
+              SEND_MESSAGES: false,
+              CREATE_PUBLIC_THREADS: false,
+              CREATE_PRIVATE_THREADS: false,
+              SEND_MESSAGES_IN_THREADS: false,
+            },
+          );
+          guildTextChannel.send("This channel has been locked by a moderator");
+        }
```

Comment:
Please revert this entire change, since functionality isn't affected we shouldn't change the code

Reply:
Done !